### PR TITLE
Add div_precision_increment system variable doc

### PR DESCRIPTION
**Pull request by yibin87**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/16813
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -506,6 +506,14 @@ For more possible values of this variable, see [Authentication plugin status](/s
 
 </CustomContent>
 
+### div_precision_increment <span class="version-mark">New in v8.0.0</span>
+
+- Scope: SESSION | GLOBAL
+- Applies to hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value): Yes
+- Default value: `4`
+- Range: `[0, 30]`
+- This variable indicates the number of digits by which to increase the scale of the result of division operations.
```
Comment (hfxsd) [inline system-variables.md:515]:
```suggestion
- Type: Integer
- Default value: `4`
- Range: `[0, 30]`
- This variable indicates the number of digits to specify the scale of a division operation result. 
```

File: system-variables.md [hunk 2]
```diff
@@ -506,6 +506,15 @@ For more possible values of this variable, see [Authentication plugin status](/s
...
 </CustomContent>
 
+### div_precision_increment <span class="version-mark">New in v8.0.0</span>
+
+- Scope: SESSION | GLOBAL
+- Applies to hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value): Yes
+- Type: Integer
+- Default value: `4`
+- Range: `[0, 30]`
+- This variable is used to control the number of decimal places in the result of a division operation performed using the operator.
```
Comment (hfxsd) [inline system-variables.md:516]:
```suggestion
- This variable is used to control the number of decimal places in the result of a division operation performed using the operator `/`.
```

File: system-variables.md [hunk 3]
```diff
@@ -506,6 +506,15 @@ For more possible values of this variable, see [Authentication plugin status](/s
 
 </CustomContent>
 
+### div_precision_increment <span class="version-mark">New in v8.0.0</span>
+
+- Scope: SESSION | GLOBAL
```
Comment (hfxsd) [inline system-variables.md:511]:
```suggestion
- Scope: SESSION | GLOBAL
- Persists to cluster: Yes
```

File: system-variables.md [hunk 4]
```diff
@@ -506,6 +506,16 @@ For more possible values of this variable, see [Authentication plugin status](/s
...
+- Persists to cluster: Yes
+- Applies to hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value): Yes
+- Type: Integer
+- Default value: `4`
+- Range: `[0, 30]`
+- This variable is used to control the number of decimal places in the result of a division operation performed using the operator `/`.
```
Comment (windtalker) [inline system-variables.md:517]: How about use the exactly describe as in mysql's doc:
```
This variable indicates the number of digits by which to increase the scale of the result of division operations performed with the `/` operator.
```

Comment (yibin87) [reply system-variables.md:517]: Sounds good, I'll change it.

**Review by windtalker — APPROVED** [review]
LGTM

File: system-variables.md [hunk 5]
```diff
@@ -506,6 +506,16 @@ For more possible values of this variable, see [Authentication plugin status](/s
...
 
+### div_precision_increment <span class="version-mark">New in v8.0.0</span>
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: Yes
+- Applies to hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value): Yes
+- Type: Integer
+- Default value: `4`
+- Range: `[0, 30]`
+- This variable indicates the number of digits by which to increase the scale of the result of a division operation performed using the `/` operator.
```
Comment (joey-yez) [inline system-variables.md:517]:
```suggestion
- This variable indicates the number of digits by which to increase the scale of the result of a division operation performed using the `/` operator. This is the same as MySQL.
```

File: system-variables.md [hunk 6]
```diff
@@ -506,6 +506,16 @@ For more possible values of this variable, see [Authentication plugin status](/s
...
 
+### div_precision_increment <span class="version-mark">New in v8.0.0</span>
+
+- Scope: SESSION | GLOBAL
+- Persists to cluster: Yes
+- Applies to hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value): Yes
+- Type: Integer
+- Default value: `4`
+- Range: `[0, 30]`
+- This variable indicates the number of digits by which to increase the scale of the result of a division operation performed using the `/` operator. This variable is the same as MySQL.
```
Comment (Oreoxmt) [inline system-variables.md:517]:
```suggestion
- This variable specifies the number of digits by which to increase the scale of the result of a division operation performed using the `/` operator. This variable is the same as MySQL.
```

**Review by Oreoxmt — APPROVED** [review]

**Review by hfxsd — APPROVED** [review]

**Review by joey-yez — APPROVED** [review]

**Pull request merged**